services/src/services/errorTelemetry/errorTelemetry.service.js

```diff
@@ -7,16 +7,34 @@
  * simply be taking the last of version records associated with the instance.
  */
 
-const createService = require('feathers-sequelize');
-const createModel = require('../../models/error_log');
+const fs             = require('fs');
+
 const hooks = require('./errorTelemetry.hooks');
 
 module.exports = function (app) {
-  const options = {
-    name: 'error_log',
-    Model: createModel(app)
-  };
+  class MyService {
+    constructor() {
+    }
+    create(data) {
+      // Should be impossible because it passed the hooks step
+      if(!data) {
+        return Promise.reject({status:'KO'});
+      }
+
+      this.storeData(data);
+
+      return Promise.resolve({status:'OK'});
+      // Called
+    }
+
+
+    storeData(data) {
+      // FIXME: TBD where, what and how to actually send data
+      const toWrite = `${new Date()} => ${JSON.stringify(data)}\n\n`;
+      fs.appendFileSync('/tmp/blah', toWrite);
+    }
+  }
 
-  app.use('/errorTelemetry', createService(options));
-  app.service('errorTelemetry').hooks(hooks.getHooks());
+  app.use('/telemetry/error', new MyService());
+  app.service('telemetry/error').hooks(hooks.getHooks());
```

Comment:
This is not referring to the same service as used on line 39. The strings are different, While I'm not sure whether Feathers is smart enough to match these two (`/telemetry/error` and `telemetry/error`) it's best to assume not :smiley_cat:

Reply:
I will check, but IIUC what you mean, I used the same _principle_ that I saw in other classes. Like in the Registration:

https://github.com/jenkins-infra/evergreen/blob/d0124a88948a35d2cfa1e86c0fbfe0a17a552362/services/src/services/registration/registration.service.js#L16-L17

I assumed it as the first line was the path, and the second was a logical name for the service. But didn't check exactly how/what this was really working under the hood.

Reply:
Ah! You're right, I had forgotten about that, and the oddity of a "service name" using a slash confused me.

Since this is consistent in other services, I wouldn't worry about it too much. When double-checking the FeathersJS documentation, it's very unclear about the "service name" versus "path." 

Let's not worry about this for now; leave this as is please